README.md

```diff
@@ -55,6 +55,36 @@ They have a lot in common, but when you have a specific problem or need, go to P
 include proxmox
 ```
 
+By default, the module creates two bridges/networks for your VMs/CTs:
+
+* vmbr0 is the public network, where you can use your additional/failover IPs for your load balancer, firewall, etc... Any public VM needs an interface here.
+* vmbr1 is the private network, for the application/database/backend VMs that don't need to be acessible directly from the internet. Connect an interface here and you get:
+  * a private IP (10.0.1.0/24) by DHCP,
+  * access to the internet through NAT,
+  * local DNS resolution so everyone can find their friends
+
+If you want to use the private network, you need at least one VM with an interface on **both** bridges to act as firewall, load balancer, VPN, SSH relay, whatever... This is the only way you can access VMs on the private network outside of a VNC console.
+
+If you want to add more bridges networks, you'll have to use [example42/puppet-network](https://github.com/example42/puppet-network/)'syntax. Here's an example:
+
+```
+# Private network bridge for super-secure VMs
+network::interface { 'vmbr2':
+  family       => 'inet',
+  address      => '10.0.2.1/24',
+  bridge_ports => ['none'],
+  bridge_stp   => 'off',
+  bridge_fd    => 0,
+  post_up      => [
```

Comment:
Maybe add that this is if the subnet needs access to other through the public bridge.